src/drawDirective.coffee

```diff
@@ -25,12 +25,13 @@ angular.module('ui-leaflet')
     scope: false
     replace: false
     require: ['leaflet']
-    controller: ($scope) ->
+    controller: ['$scope', ($scope) ->
```

Comment:
I couldn't stand it, I had to make that test harness again :smile: . I've created it as a separate branch in my repo but you you'd like me to add it to this PR, I'll happily do so.

See the test harness at https://github.com/tomsaleeba/ui-leaflet-draw/tree/issue-8-tester/minification-test